packages/cyberstorm/src/components/LinkLibrary.tsx

```diff
@@ -0,0 +1,63 @@
+import {
+  LinkLibrary,
+  thunderstoreLinkProps,
+  ThunderstoreLinkProps,
+} from "@thunderstore/components";
+import NextLink from "next/link";
+import React, { forwardRef } from "react";
+
+interface CyberstormLinkProps {
+  children?: React.ReactNode;
+}
+
+interface LinkProps extends CyberstormLinkProps, ThunderstoreLinkProps {
+  queryParams?: string;
+  url: string;
+}
+
+const CyberstormLink = forwardRef<HTMLAnchorElement>(
+  (props: CyberstormLinkProps, ref) => {
+    const { ...rest } = props;
+
+    return (
+      // eslint-disable-next-line jsx-a11y/anchor-has-content
+      <a ref={ref} {...rest} />
+    );
+  }
+);
+
+CyberstormLink.displayName = "CyberstormLink";
+
+const Link = (props: LinkProps): React.ReactElement => {
+  const { children, queryParams, url, ...cyberStormProps } = props;
+  const q = queryParams ? `?${queryParams}` : "";
+
+  for (const key in thunderstoreLinkProps) {
+    delete cyberStormProps[key as keyof ThunderstoreLinkProps];
+  }
+
```

Comment:
What do you think `cyberstormProps` variable contains at this row?

Reply:
If I understood correctly these 3 rows delete the "unwanted" prop types from the `cybeStormProps` const 🤔

Let me know if I'm correct, but as with the previous review comments, files now deleted 👍

Reply:
IIRC the issue was that you passed like one prop so this did nothing.